bloom-housing/bloom
# 1589/Sets Redis to handle backend cache

## Issue

Addresses # #1589 


## Description

This sets NestJS cache manager to use Redis instead of in memory.

## Type of change


## How Can This Be Tested/Reviewed?

Spin up both partner and portal apps. You can monitor Redis with `redis-cli monitor`. When you edit a listing in partners you should see the cache entry get cleared. When you go to listings on public, you should see the entry get added.


## Checklist:
